core/frontend/src/components/vehiclesetup/overview/PowerInfo.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <v-card class="ma-2 pa-2">
+    <v-card-title class="align-center">
+      Battery Monitor
+    </v-card-title>
+    <v-card-text>
+      <div class="d-flex justify-space-between mb-3">
+        <span><b>Voltage:</b> {{ battery_voltage }} V</span>
+        <span><b>Current:</b> {{ battery_current }} A</span>
+      </div>
+
+      <div v-if="is_known_power_meter">
+        <b>{{ is_known_power_meter }}</b>
+      </div>
+      <div
+        v-else
+      >
+        <span class="d-block"><b>Voltage Pin:</b> {{ printParam(batt_monitor.vPin) }}</span>
+        <span class="d-block"><b>Voltage Multiplier:</b> {{ printParam(batt_monitor.vMulti) }}</span>
+        <span class="d-block"><b>Current Pin:</b> {{ printParam(batt_monitor.aPin) }}</span>
+        <span class="d-block"><b>Current per Volt:</b> {{ printParam(batt_monitor.aPv) }}</span>
+        <span class="d-block"><b>Current Offset:</b> {{ printParam(batt_monitor.aOff) }}</span>
+      </div>
+      <span class="d-block"><b>Low Voltage Failsafe:</b> {{ printParam(batt_failsafe_low.action) }}</span>
+      <span
+        v-if="batt_failsafe_low.level?.value"
+        class="d-block"
+      >
+        <b>Low Voltage Level:</b> {{ printParam(batt_failsafe_low.level) }}
+      </span>
+      <span class="d-block"><b>Critical Voltage Failsafe:</b> {{ printParam(batt_failsafe_critical.action) }}</span>
+      <span
+        v-if="batt_failsafe_critical.level?.value"
+        class="d-block"
+      >
+        <b>Low Voltage Level:</b> {{ printParam(batt_failsafe_low.level) }}
+      </span>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+import autopilot_data from '@/store/autopilot'
+import mavlink from '@/store/mavlink'
+import { printParam } from '@/types/autopilot/parameter'
+import mavlink_store_get from '@/utils/mavlink'
+
+const power_modules_list = [
```

Comment:
It would be nice to add a reference for 
https://github.com/mavlink/qgroundcontrol/blob/d1c850c6e8efececf9c41861ba9077ab82f4efd3/src/AutoPilotPlugins/APM/APMPowerComponent.qml#L313